assets/gabe-idle-run-animation.trickfilm.ron

```diff
@@ -1,12 +1,11 @@
 {
     "idle": (
         keyframes: KeyframesVec([0]),
-        /* optional */ keyframe_timestamps: Some([0.0]), /* Must be same size as keyframes, if provided */
+        /* optional */ keyframe_timestamps: [0.0], /* Must be same size as keyframes, if provided */
```

Comment:
You changed the whole keyframe timestamps implementation in this PR, was that because it was necessary for the events to work? I like this change though.

Reply:
I had to rewrite the whole Deserialization manually. That way I got rid of a lot of additional types. I also changed the way optional parameters should be handled.
I am not yet sure what I will go ahead with. Need to look into some other rust projects on how they handle optional fields.